src/DetailsPage/Documents/Document.js

```diff
@@ -37,33 +37,36 @@ class Document extends Component {
     }
   };
 
-  render() {
-    const { document } = this.state;
 
-    if (!document) return <React.Fragment />
-
-    return (
-      <TableRow key={document.name}>
-        <TableColumn>
-          <a
-            className={`icon ${this.getDocumentIcon(document)}`}
-            href={document.downloadURL}
-            target="_blank"
-            rel="noopener noreferrer"
-          >
-            {document.name}
-          </a>
-        </TableColumn>
-        <TableColumn className="md-text-right">
-          {document.hashMatch && document.sizeMatch ? (
-            <FontIcon secondary>done</FontIcon>
-          ) : (
-            <FontIcon error>block</FontIcon>
-          )}
-        </TableColumn>
-      </TableRow>
-    );
-  }
+  return (
+    <>
+      {
+        !document ?
+          <React.Fragment />
```

Comment:
`<>` and `<React.Fragment>` are the same thing in React. Please be consistent and use either the one or another, but not both within the same component